src/components/board/PostWriteForm.tsx

```
@@ -0,0 +1,228 @@
+                onClick={() => setAnonymous((v) => !v)}
+                className={`mr-1 flex items-center gap-1.5 text-sm transition-colors ${
+                  anonymous ? "text-gray-800" : "text-gray-400 hover:text-gray-600"
+                }`}
+              >
+                <span
+                  className={`flex h-4 w-4 shrink-0 items-center justify-center rounded border transition-colors ${
+                    anonymous ? "border-gray-800 bg-gray-800" : "border-gray-300"
+                  }`}
+                >
+                  {anonymous && (
+                    <svg width="10" height="8" viewBox="0 0 10 8" fill="none">
```

Biome (2.5.0)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)